### PR TITLE
VSCODE-163: Add resources/help panel

### PR DESCRIPTION
**Pull request by Anemy**
This PR adds a Help and Feedback panel with links to different resources:
Screen Shot 2020-09-08 at 6 57 18 PM
Screen Shot 2020-09-08 at 6 57 12 PM

Open qs:
- For reporting bugs do we want to link them to github or jira?

Tweaking the icons still a bit with Claudia, figured I'd open the PR now since its mostly styles.

### CODE REVIEW

File: src/explorer/playgroundsTree.ts [hunk 1]
```diff
@@ -149,9 +149,9 @@ export default class PlaygroundsTree
     return element;
   }
 
-  private getFileNames(path: string): Promise<string[]> {
+  private getFileNames(filePath: string): Promise<string[]> {
```
Comment (alenakhineika) [inline src/explorer/playgroundsTree.ts:152]: 😏👍

**Review by alenakhineika — COMMENTED** [review]
Yeah, it would be nice to play with icon's sizes. Now also because they have a different shape, they look sometimes much bigger than text, or very small like MongoDB leaf. Looking forward to seeing the final version! This help panel should be very useful for users.

File: src/explorer/playgroundsTree.ts [hunk 2]
```diff
@@ -16,10 +16,10 @@ export class FileStat implements vscode.FileStat {
...
       ? vscode.FileType.File
       : this.fsStat.isDirectory()
-      ? vscode.FileType.Directory
-      : this.fsStat.isSymbolicLink()
-      ? vscode.FileType.SymbolicLink
-      : vscode.FileType.Unknown;
+        ? vscode.FileType.Directory
+        : this.fsStat.isSymbolicLink()
+          ? vscode.FileType.SymbolicLink
+          : vscode.FileType.Unknown;
```
[the remark this answers is not in the document]
Comment (Anemy) [reply src/explorer/playgroundsTree.ts:22]: Eslint is doing this one - it's also complaining that it's too many layers of ternary 😅

File: src/explorer/helpExplorer.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      this._treeController.activateTreeViewEventHandlers(
+        this._treeView,
+        telemetryController
+      );
+    }
+  }
+
+  deactivate(): void {
+    if (this._treeView) {
+      this._treeView.dispose();
```
[the remark this answers is not in the document]
Comment (Anemy) [reply src/explorer/helpExplorer.ts:31]: From what I understand, disposing cleans up the listeners we create in `activateTreeViewEventHandlers` (`treeView.onDidChangeSelection`)
https://vscode-docs.readthedocs.io/en/stable/extensions/patterns-and-principles/#disposables
I think it's something we want to keep, just to make sure we clean up a bit.

**Comment by alenakhineika** [comment]
Tested links and collapsing state, everything works well! When tests are fixed and icons updated can be merged 👌

**Review by alenakhineika — APPROVED** [review]

**Pull request merged**